templates/helm/templates/service-account.yaml

```
@@ -10,8 +10,9 @@ metadata:
     k8s-app: {{ include "app.name" . }}
     helm.sh/chart: {{ include "chart.name-version" . }}
   name: {{ include "service-account.name" . }}
+  namespace: {{ "{{ .Release.Namespace }}" }}
```

why add the quotes here? In all the other files, the value is just `{{ .Release.Namespace }}` . Best to keep it consistent with all other template files.
```suggestion
  namespace: {{ .Release.Namespace }}
```

---

Oh boy, I just copied that namespace line from the wrong place, without noticing that the source was actually a template for a template. :see_no_evil: